packages/notification-email-sender/test/sendEserviceDescriptorPublishedSimpleEmail.integration.test.ts

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable no-irregular-whitespace */
+import fs from "fs/promises";
+import path from "path";
+import { fileURLToPath } from "url";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  getMockAgreement,
+  getMockDescriptor,
+  getMockEService,
+  getMockTenant,
+  getMockTenantMail,
+} from "pagopa-interop-commons-test";
+import {
+  AgreementId,
+  Descriptor,
+  EService,
+  Tenant,
+  generateId,
+  tenantMailKind,
+  toEServiceV2,
+} from "pagopa-interop-models";
+import { describe, expect, it, vi } from "vitest";
+import axios, { AxiosResponse } from "axios";
+import { eventMailTemplateType } from "../src/services/notificationEmailSenderService.js";
+import {
+  addOneAgreement,
+  addOneEService,
+  addOneTenant,
+  notificationEmailSenderService,
+  interopFeBaseUrl,
+  sesEmailManager,
+  sesEmailManagerConfig,
+  sesEmailsenderData,
+  templateService,
+} from "./utils.js";
+
+describe("sendEserviceDescriptorPublishedSimpleEmail", () => {
+  it("should send an email on EserviceDescriptorPublished", async () => {
```

Comment:
```suggestion
  it("should send an email on EserviceDescriptorPublished when there're active agrrements", async () => {
```
in this test we should add some agreements in not active states, in order to be sure that only the one with Active, Suspended and Pending receive the mail